config/foreman-proxy-content.migrations/191218160614-update-hook-dirs.rb

```diff
@@ -0,0 +1,2 @@
+scenario[:hook_dirs] = ['/usr/share/foreman-installer/katello/hooks'] unless scenario[:hook_dirs].include?('/usr/share/foreman-installer/katello/hooks')
+scenario[:installer_dir] = '/usr/share/foreman-installer/katello' unless scenario[:installer_dir] == '/usr/share/foreman-installer/katello'
```

Comment:
@ekohl I am blanking a bit on how to handle these migrations given we need it to migrate to update the full-path but the tests will fail because it compares the pre and post migrated states to current config file which has relative paths.

Reply:
How about:
```ruby
if File.basename(scenario[:installer_dir]) == 'katello'
  scenario[:hook_dirs] = [File.join(scenario[:installer_dir], 'hooks')]
  scenario[:installer_dir] = File.dirname(scenario[:installer_dir])
end
```